src/main/java/com/elixr/poc/rest/response/DeleteResponse.java

```diff
@@ -0,0 +1,9 @@
+package com.elixr.poc.rest.response;
+
+import lombok.Data;
+
+@Data
+public class DeleteResponse {
+    private boolean success;
+    private String message;
```

Comment:
It should be errorMsg. Also, could you use Jackson annotations to exclude parameters and change the parameters' label in the response?

Reply:
Updated.